presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveQueries.java

```diff
@@ -507,6 +507,39 @@ public void testStringFunctions()
         assertQuery("SELECT comment, reverse(comment) FROM orders");
     }
 
+    @Test
+    public void testBinaryFunctions()
+    {
+        // Crc32.
```

Comment:
Nit -- Change all the functions in the comments to start with lowercase letters. e.g. crc32 instead of Crc32.

Reply:
Sure, I will do that. I just followed the existing tests that use capitalised function names, e.g. Reverse, Substr.
No problem, I will update.

Reply:
Thanks! Updated!